File: prime.java
public class prime {

    public static void main(String[] args)
 {
        System.out.println("Prime numbers from 1 to 100 are:");

        for (int i = 2; i <= 100; i++)
 { 
            if (isPrime(i)) 
{
                System.out.print(i + " ");
            }
        }
    }

    
    public static boolean isPrime(int number)
 {
        if (number <= 1) 
{ 
            return false;
        }
       
        
        for (int i = 2; i * i <= number; i++)
 { 
            if (number % i == 0) 
{
                return false; 
            }
        }
        return true; 
    }
}

File: prime.py
import math

def is_prime(num):
    """
    Checks if a number is prime.
    A prime number is a natural number greater than 1 that has no positive divisors other than 1 and itself.
    """
    if num <= 1:
        return False  
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            return False  
    return True  

print("Prime numbers from 1 to 100 are:")
for number in range(1, 101):
    if is_prime(number):
        print(number)